ToolBox/CoinApiDataConverter/CoinApiBitfinexSymbolMapper.cs

```diff
@@ -0,0 +1,99 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Brokerages;
+
+namespace QuantConnect.ToolBox.CoinApiDataConverter
+{
+    /// <summary>
+    /// Mapper of CoinApi Bitfinex symbols with more than 3 characters.
+    /// </summary>
+    /// <seealso cref="QuantConnect.Brokerages.ISymbolMapper" />
+    public class CoinApiBitfinexSymbolMapper : ISymbolMapper
+    {
+        private HashSet<Tuple<string, string>> _mappedSymbols = new HashSet<Tuple<string, string>>
+        {
+            new Tuple<string, string>("anio", "nio"),
+            new Tuple<string, string>("bchsv", "bsv"),
+            new Tuple<string, string>("dash", "dsh"),
+            new Tuple<string, string>("iota", "iot"),
+            new Tuple<string, string>("mana", "mna"),
+            new Tuple<string, string>("pkgo", "got"),
+            new Tuple<string, string>("qtum", "qtm"),
+            new Tuple<string, string>("usdt", "ust"),
+            new Tuple<string, string>("yoyow", "yyw")
+        };
+
+        /// <summary>
+        /// Converts a Lean symbol instance to a brokerage symbol
+        /// </summary>
+        /// <param name="symbol">A Lean symbol instance</param>
+        /// <returns>
+        /// The brokerage symbol
+        /// </returns>
+        public string GetBrokerageSymbol(Symbol symbol)
+        {
+            var leanSymbol = symbol.Value;
+
+            var pair = new[] { leanSymbol.Substring(0, 3), leanSymbol.Substring(3, 3) };
```

Comment:
Maybe aligned with the changes of the PR https://github.com/QuantConnect/Lean/pull/3000 we could use the `SymbolPropertiesDatabase` to get the quote and base currency? @jaredbroad what do you guys think?

Reply:
Since here we don't need all symbol properties (tick sizes, tick increments, etc.) but only the ticker mapping, I think it's easier to maintain with a simple REST API call.
I just pushed an update for this.